frequency               : 5              # [hz]

pc_topic                : points
costmap_topic           : local_costmap

base_link_frame_id      : base_link

width                   : 20          # [m]
resolution              : 0.2         # [m]

height_min              : 0.0         # [m]
height_max              : 1.0         # [m]